fast-context-generic/src/createFastContext.tsx

```diff
@@ -55,8 +55,10 @@ export default function createFastContext<Store>(initialState: Store) {
       throw new Error("Store not found");
     }
 
-    const state = useSyncExternalStore(store.subscribe, () =>
-      selector(store.get())
+    const state = useSyncExternalStore(
+      store.subscribe,
+      () => selector(store.get()),
+      () => selector(store.get()),
```

Comment:
I think it should be `() => initialState` since the initial state is what we get from the server and in the docs for this hook the requirement is that you always return the same reference from `getServerSnapshot`.

Reply:
Or should it be `() => selector(initialState)`  to get the actual field?

Reply:
Thanks you're right. Should it be `() => selector(initialState)`?

Reply:
Ah, yes, true, the selector needs to be there. Alright, good fix, let's add it.

Reply:
Fixed the PR. Thank you!